geode-core/src/main/java/org/apache/geode/internal/cache/EntryEventImpl.java

```diff
@@ -165,6 +165,9 @@
    */
   private byte[] deltaBytes = null;
 
+  /* Is this special about recalculating size? */
```

Comment:
Can this comment be removed?

Reply:
Well, it probably shouldn't be removed, since all the other fields have comments as well. However, it can be improved.